libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -143,6 +144,21 @@ class UdpContext
         return _rx_buf->len - _rx_buf_offset;
     }
 
+    size_t tell() const
+    {
+        return _rx_buf_offset;
+    }
+
+    void seek(const size_t pos)
+    {
+        assert(isValidOffset(pos));
+        _rx_buf_offset = pos;
+    }
+
+    bool isValidOffset(const size_t pos) const {
+        return (pos <= _rx_buf->len);
```

Comment:
Thanks.
Since you are going to use this offset, would'nt it be ```pos < _rx_buf->len``` ?
Sorry I don't mean to be pushy :]

Reply:
Not at all, I appreciate your comments, I just havn't had much time to do more than fix them during breaks at work today =D
The check on line 222 in the 'read' function checks if the offset is at least equal to the limit, which should give that to set the buffer as "fully read", the offset should be set just past the limit, i.e. equal to the length. I figured "finished reading" was a valid use-case and thus a valid offset.

Reply:
Thanks for the explanation